content/docs/install/istio/index.md

```diff
@@ -1,11 +1,16 @@
 ---
-title: Expose deployed services
+title: Install and configure Istio
 description: Configure Istio and create a ConfigMap to expose services deployed by Keptn.
-weight: 1
-keywords: [0.17.x-cd]
+weight: 75
 ---
 
-Istio must be installed in order to access the services that Keptn deploys. This means that the `istio-ingressgateway` service should already be available in the `istio-system` namespace and a `public-gateway` need to be created as explained below. In addition, a ConfigMap must be edited that tells Keptn how the gateway is configured.
+Istio can be installed in order to access the services that Keptn deploys.
+You can instead use and install the [Job Executor Service](../jes-install).
+
+If you are using Istio for this,
+that the `istio-ingressgateway` service must already be available in the `istio-system` namespace
+and a `public-gateway` need to be created as explained below.
+In addition, a ConfigMap must be edited that tells Keptn how the gateway is configured.
```

Comment:
We have helm values for that:
https://github.com/keptn/keptn/blob/master/installer/manifests/keptn/values.yaml#L3-L7

Reply:
I think this is actually what is documented in the section below but the existing prose doesn't make that very clear.  I did some minor rewording for now; we should revisit this whole section more later.

Reply:
Yes, and I think they are actually what is documented later in this page, but this intro certainly does not convey that!  I reworded it a bit but we need to come back and look at all of this a little more.